include/ibm/locks.hpp

```diff
@@ -209,23 +209,12 @@ inline RcGetLockList Lock::getLockList(const ListOfSessionIds& listSessionId)
 inline RcReleaseLockApi Lock::releaseLock(const ListOfTransactionIds& p,
                                           const SessionFlags& ids)
 {
-
-    bool status = validateRids(p);
```

Comment:
What does isItMyLock do if the lock doesn't exist? 
I would think validating the locks is still a valid operation?

Reply:
@gtmills I re-added the validateRids() function in my last patch set. The function isItMyLock() will return false if the lock (transaction id) is not owned by that session id

Reply:
Ack